### DISCUSSION TITLE
Confirming values in the cell2location object are cell type abundances

### DISCUSSION THREAD

**Discussion by ChristinaSteyn** [BayraktarLab/cell2location#257] [Q&A]
Hi there, 

A bit of an obvious question but I just wanted to confirm that the following values in the cell2location represent the abundances of each cell type? I ran the code below on the adata_vis object after running through the standard tutorial (https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html).

n = adata_vis.uns["mod"]["factor_names"]
test = adata_vis.obs[n]
![cell2location](https://user-images.githubusercontent.com/83014373/223173176-2e46b3f7-1459-43bf-af5b-f4b4c1b71177.png)

Thanks in advance!

**Comment by vitkl** [comment]
The answer to your question is yes, the reported numbers are the expected abundance of each cell type at every location.

**Reply by ChristinaSteyn** [reply]
Thanks very much!
